packages/swapper/src/swappers/cow/cowGetTradeTxs/cowGetTradeTxs.ts

```diff
@@ -16,20 +22,21 @@ export async function cowGetTradeTxs(deps: CowSwapperDeps, input: TradeResult):
     } = getOrdersResponse
 
     if (status !== ORDER_STATUS_FULFILLED) {
-      return {
+      return Ok({
         sellTxid: '',
-      }
+      })
     }
 
     const getTradesResponse = await cowService.get<CowSwapGetTradesResponse>(
       `${deps.apiUrl}/v1/trades/?orderUid=${input.tradeId}`,
     )
 
-    return {
+    return Ok({
       sellTxid: input.tradeId,
       buyTxid: getTradesResponse.data[0].txHash,
-    }
+    })
   } catch (e) {
+    // TODO(gomes): don't throw in this module
```

Comment:
Should we be returning `Err` types here, or is that out of scope for this PR?

Reply:
We definitely should, now that we expect a `Result` - though our web consumption luckily has catch clauses so this would accidentally work as-is. Taking another look at this, we should at minimum return `Err()` in all catch clauses. Tackled in 09a28a51836f5420d560b530e3529b9c03e9c248